### ISSUE TITLE
Since last version 14.03 the layer view is separated in 2 peaces and the top part is shown with a different x/y position

### ISSUE THREAD

**Issue by frankjoke** [daid/LegacyCura#796]
The top part of each print when viewed in layer mode is off. It happens to every print, but normal gcode is OK.
See below:

![2014-03-22_234239](https://f.cloud.github.com/assets/6932853/2492118/886fa4fe-b213-11e3-9bd1-5e75fabc9886.png)

I could not find reason for it and it did not go away (unless goning back to old 14.01).

**Comment by Dim3nsioneer** [comment]
I have the same effect in y direction. In my case, it comes from a G0 command followed by a G92. This means resetting coordinates with G92 seems to have this effect.

**Comment by frankjoke** [comment]
Interesting observation but I do not enter (or check) any G-Code generated by Cura.
Anyhow, I played a bit and realized it is alway the top 10 layers which are off.
If you move the slider down the layers you see that the top 10 are always displayed with an offset.
At least with me :(

**Comment by Dim3nsioneer** [comment]
If I remember correctly, there is at least one G92 in the standard start.gcode of Cura. Of course this cannot be the reason if you have an UM2.
I'm positive that we will get some more information from Daid tomorrow.

**Comment by daid** [comment]
Yeah, this is related to the start code / G92. The lower layers are not generated from the gcode but straight from the slicer info, causing the offset right now. The G92 is generating an wrong offset in the layerview anyhow, so I still need to fix that.

**Comment by PaulHam211** [comment]
Is there any way to fix this? i get the same issue on layer view, the gcode prints find on the printer
